src/main/java/ke/co/proxyapi/jobnotificationparser/utils/AppUtils.java

```diff
@@ -35,4 +53,53 @@ public void sendToTelegram(String urlStr)
 			template.asyncSendBody("direct:telegram", advert);
 		}
 	}
+
+	private boolean hasSpamWords(String url) {
+		String description = getMetaDescription(url);
+		if (description == null) {
+			return false;
+		}
+		return isSpamDescription(description, "spam_words.txt");
+	}
+	public static String getMetaDescription(String url) {
+		try {
+			Document doc = Jsoup.connect(url).get();
+			Elements metaTags = doc.getElementsByTag("meta");
+			for (Element metaTag : metaTags) {
+				String tagName = metaTag.attr("name");
+				if (tagName.equalsIgnoreCase("description")) {
+					return metaTag.attr("content");
```

Comment:
Did you confirm this tag actually contains the words we want to filter later? Did a few checks, and so far the spam words are not present in some of their meta descriptions